client/js/jsonedit.js

```diff
@@ -2072,8 +2097,8 @@ function jeShowCommands() {
     if(command.context == undefined) {
       const name = (typeof command.name == 'function' ? command.name() : command.name);
       const icon = (typeof command.icon == 'function' ? command.icon() : command.icon);
-      const isMaterial = String(icon).match(/^[^[]/) ? 'material' : '';
-      commandText += `<button class='top ${isMaterial}' id='${command.id}' title='${name}' ${!command.show || command.show() ? '' : 'disabled'}>${icon}</button>`;
+      const classes = typeof command.classes == 'function' ? command.classes() : command.classes || '';
+      commandText += `<button class='top ${classes}' id='${command.id}' title='${name}' ${!command.show || command.show() ? '' : 'disabled'}>${icon}</button>`;
```

Comment:
Hmm, I think I would prefer to still detect `material` automatically and then append the result of `command.classes()`. What do you think?